TeaDiary/Scripts/App/TeaListDisplayer.js

```diff
@@ -1,17 +1,24 @@
 ﻿function TeaListDisplayer(container) {
     var self = this;
     self.container = $(container);
-
+    var teaListTable = $("<table>");
+    var teaListContainer = $("<tbody>");
+    var entityCounter = 1;
+    
     self.display = (teas) => {
         self.clearContainer();
         setContainerLoading(true);
-
-        var header = buildHeader(teas);
-        appendElement(header);
+        
+        displayHeader(teas);
         teas.forEach((tea) => {
             displayTea(tea);
```

Comment:
Why don't you `setupTeaTable()` first and then `displayXxx()` your stuff? It will be much easier to follow